pkgs/tools/compression/bzip2/default.nix

```diff
@@ -1,5 +1,6 @@
 { lib, stdenv, fetchurl
-, linkStatic ? with stdenv.hostPlatform; isStatic || isCygwin
+, enableStatic ? with stdenv.hostPlatform; isStatic || isCygwin
+, disableShared ? false
```

Comment:
Do we need to assert that not both are disabled?

Reply:
> I think we should name them both either enable or disable to be less confusing

Done.

> Do we need to assert that not both are disabled?

Probably not; currently that would result in no special flags being passed to `./configure`, which is probably OK and maybe handy for somebody sometime.